test/tools/router_check/BUILD

```diff
@@ -22,6 +22,8 @@ envoy_cc_test_library(
         "router.cc",
         "router.h",
     ],
+    copts = ["-DHAVE_LONG_LONG"],
```

Comment:
I took it from an [example](https://github.com/envoyproxy/envoy/blob/668725e7035e4b3f655222b2e595558ca9b0482b/source/server/BUILD#L201).Is this the right `-copts` to add?